src/Covid19Dashboard/PlotChart/index.jsx

```diff
@@ -62,42 +110,55 @@ class PlotChart extends PureComponent { // eslint-disable-line react/no-multi-co
 
   render() {
     const chartData = formatChartData(this.props.plots);
+    const { width } = this.state;
 
     return (
       <div className='plot-chart'>
         <p className='plot-chart__title'>
           {this.props.title}
         </p>
-        <ResponsiveContainer>
+        <ResponsiveContainer height={250}>
           <LineChart
             data={chartData.data}
             margin={{
               top: 5, right: 30, left: 20, bottom: 5,
             }}
           >
-            <CartesianGrid strokeDasharray='3 3' />
+            <CartesianGrid
+              horizontal={true}
+              vertical={false}
+              strokeDasharray='3 3'
+            />
             <XAxis
               dataKey='date'
               // label={this.props.xTitle}
               tick={<CustomizedAxisTick />}
-              interval={1}
+              ticks={chartData.ticks}
+              // interval='preserveStartEnd'
+              interval={0}
             />
             <YAxis
               label={{
                 className: 'plot-chart__y-title',
                 value: this.props.yTitle,
                 angle: -90,
                 position: 'insideLeft',
+                offset: -10
               }}
               type='number'
               domain={[0, 'auto']}
             />
             <Tooltip content={this.renderTooltip} />
-            <Legend />
-            <Line type='monotone' dataKey={this.props.plots[0].name} stroke='#8884d8' dot={false} />
-            <Line type='monotone' dataKey={this.props.plots[1].name} stroke='#aa5e79' dot={false} />
+            <Legend onMouseEnter={this.handleMouseEnter} onMouseLeave={this.handleMouseLeave} />
+            {
+              this.props.plots.map((entry, index) => {
+                const color = COLORS[index];
+                return <Line key={entry.name} type='monotone' dataKey={entry.name} strokeWidth={width[entry.name]} stroke={color} dot={false} />;
+              })
+            }
           </LineChart>
         </ResponsiveContainer>
+        <div style={{ marginTop: `${2}em` }}><p>{this.props.description}</p></div>
```

Comment:
we talked about the description being displayed in a popup when hovering over the chart - if you don't plan to fix it in this ticket, please create a new ticket?

Reply:
I have moved it to the tooltip. We can check how it looks like in QA.

Reply:
i don't think this is what we want, but it's not deployed in QA so can't see what it looks like right now. 
iirc the description should be separate from the tooltip, it should be displayed when you hover on the chart even when you are not hovering over the plot itself. Gina can probably describe the expectation better

Reply:
will be done in ticket COV-120